app-shells/direnv/direnv-2.14.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Unclutter your .profile"
+DESCRIPTION="direnv is an environment switcher for the shell. It knows \
+how to hook into bash, zsh, tcsh and fish shell to load or unload \
+environment variables depending on the current directory. This allows \
+project-specific environment variables without cluttering the ~/.profile file."
+HOMEPAGE="https://direnv.net/"
+COPYRIGHT="2017 zimbatm and contributors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/direnv/direnv/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="917838827cb753153b91cb2d10c0d7c20cbaa85aa2dde520ee23653a74268ccd"
+PATCHES="
+	direnv-$portVersion.patchset
+	"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	direnv = $portVersion
+	cmd:direnv = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
```

Comment:
No need for a newline here per the [Guidelines][0]

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering